src/components/structures/ScrollPanel.js

```diff
@@ -332,28 +385,34 @@ module.exports = React.createClass({
 
             var boundingRect = node.getBoundingClientRect();
             if (boundingRect.bottom < wrapperRect.bottom) {
-                return {
-                    atBottom: atBottom,
+                this.scrollState = {
+                    atBottom: false,
```

Comment:
I've changed the semantics of `atBottom` slightly - whereas previously it meant simply "we're scrolled to the bottom of the timeline", now it means "we're scrolled to the bottom of the timeline *and* we should stay there on updates" - ie, `atBottom` is only set if `stickyBottom` is set. This correlates to the removal of the `stickyBottom` guard in `_restoreSavedScrollState`.

I guess a rename of `atBottom` to `stuckToBottom` would help clarify this. I'll make it so.